website/docs/docs/cloud/cloud-cli-installation.md

```diff
@@ -155,13 +155,13 @@ If you already have dbt Core installed, the dbt Cloud CLI may conflict. Here are
   - Uninstall the dbt Cloud CLI using the command: `pip uninstall dbt`
   - Reinstall dbt Core using the following command, replacing "adapter_name" with the appropriate adapter name:
     ```shell
-    pip install dbt-adapter_name --force-reinstall
+    python -m pip install dbt-adapter_name --force-reinstall
     ```
-    For example, if I used Snowflake as an adapter, I would run: `pip install dbt-snowflake --force-reinstall`
+    For example, if I used Snowflake as an adapter, I would run: `python -m pip install dbt-snowflake --force-reinstall`
 
 --------
 
-Before installing the dbt Cloud CLI, make sure you have Python installed and your virtual environment venv or pyenv . If you already have a Python environment configured, you can skip to the [pip installation step](#install-dbt-cloud-cli-in-pip).
+Before installing the dbt Cloud CLI, make sure you have Python installed and your virtual environment venv or pyenv . If you already have a Python environment configured, you can skip to the [python -m pip installation step](#install-dbt-cloud-cli-in-pip).
```

Comment:
Although I'm guessing the replacement wasn't intentional here, it wouldn't harm much. But I'm guessing we want to keep it as: 
```suggestion
Before installing the dbt Cloud CLI, make sure you have Python installed and your virtual environment venv or pyenv . If you already have a Python environment configured, you can skip to the [pip installation step](#install-dbt-cloud-cli-in-pip).
```